source/_integrations/plugwise.markdown

```
@@ -89,151 +89,143 @@ This integration follows standard integration removal. No extra steps are requir
-script:
-  reboot_gateway:
-    sequence:
-      - action: button.press
-        target:
-          entity_id: button.adam_reboot
-```
+Available action: `select.select_option`
 
-#### Set HVAC mode
+{% tip %}
+The available schedules depend on the [schedules](#schedule_management) you have configured.
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)